### PR TITLE
Fix Relative links and allow use of WG_PATH variable

### PR DESCRIPTION
**Pull request by jdeath**
This PR fixes the relative links (same as #125) and also allows use of the WG_PATH variable. The system will properly load and use config files not saved in /etc/wireguard/

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -43,4 +43,4 @@ jobs:
       with:
         push: true
         platforms: linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64/v8
-        tags: weejewel/wg-easy:latest, weejewel/wg-easy:${{ env.RELEASE }}
+        tags: jdeath/wg-easy:latest, jdeath/wg-easy:${{ env.RELEASE }}
```
Comment (WeeJeWel) [inline .github/workflows/deploy.yml:46]: ?

File: src/lib/WireGuard.js [hunk 2]
```diff
@@ -54,8 +54,8 @@ module.exports = class WireGuard {
         }
 
         await this.__saveConfig(config);
-        await Util.exec('wg-quick down wg0').catch(() => { });
-        await Util.exec('wg-quick up wg0');
+        await Util.exec(`wg-quick down ${WG_PATH}/wg0.conf`).catch(() => { });
+        await Util.exec(`wg-quick up ${WG_PATH}/wg0.conf`);
```
Comment (WeeJeWel) [inline src/lib/WireGuard.js:58]: Why is this useful? We're using Docker so who cares about the path :)

File: src/www/index.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 <head>
   <title>WireGuard</title>
-  <link href="/css/vendor/tailwind.min.css" rel="stylesheet">
+  <link href="css/vendor/tailwind.min.css" rel="stylesheet">
```
Comment (WeeJeWel) [inline src/www/index.html:6]: I prefer `./` to indicate it's a relative path.

**Review by WeeJeWel — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (jdeath) [reply .github/workflows/deploy.yml:46]: This was so I could use and test it. I didn't know how to remove this file...

File: src/lib/WireGuard.js (hunk 2 above)
Comment (jdeath) [reply src/lib/WireGuard.js:58]: I am using the container inside homeassistant (stripped down docker). I cannot mount another directory to /etc/wireguard/.

File: src/www/index.html (hunk 3 above)
Comment (jdeath) [reply src/www/index.html:6]: I can make the change. It worked, so I did it.

**Comment by WeeJeWel** [comment]
Relative links are now added.

Honestly I have no interest in WG_PATH, because it's mounted using Docker anyway.

**Pull request closed without merging**